rotkehlchen/chain/ethereum/decoding/base.py

```diff
@@ -131,7 +131,7 @@ def decode_erc20_721_transfer(
         amount_raw_or_token_id = hex_or_bytes_to_int(tx_log.data)
         if token.token_kind == EvmTokenKind.ERC20:
             amount = token_normalized_value(token_amount=amount_raw_or_token_id, token=token)
-            if event_type == HistoryEventType.SPEND:
+            if verb == 'Send':
```

Comment:
I believe it is better to avoid direct string compassion like this if we can. One of the reasons is that we can change the text in the future and this will be harder to find or if we introduce translations as codes.

Looking at the code it appears that `Send` appears when it is a `TRANSFER` or a `SPEND` so why not changing it to

```suggestion
            if event_type == (HistoryEventType.SPEND, HistoryEventType.TRANSFER):
```